src/transformers/models/roformer/tokenization_roformer.py

```diff
@@ -0,0 +1,126 @@
+# coding=utf-8
+# Copyright junnyu and The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tokenization classes for RoFormer."""
+from ...utils import logging
+from ..bert.tokenization_bert import BertTokenizer, BasicTokenizer
+import jieba
```

Comment:
This should be imported in the `__init__` and raise a nice error message if it's not installed as it's not a required dep.